server/src/LspServer.ts

```diff
@@ -69,9 +64,10 @@ export class LspServer {
   manifestParser = new ManifestParser();
   featureFinder = new FeatureFinder();
   initStart = performance.now();
-  zetaSqlWrapper = new ZetaSqlWrapper();
   onGlobalDbtErrorFixedEmitter = new Emitter<void>();
 
+  bigQueryContext: BigQueryContext;
```

Comment:
Can we delete line
`dbtProfileCreator = new DbtProfileCreator(this.yamlParser);`?

Reply:
Fixed.